lib/commons/loading.dart

```diff
@@ -6,19 +6,32 @@ import 'package:flutter_spinkit/flutter_spinkit.dart';
 class Loading extends StatelessWidget {
 
   final double size;
+  final Widget loader;
 
-  Loading({this.size = 30});
+  Loading({this.size = 30, this.loader});
 
   @override
   Widget build(BuildContext context) {
-    return new SpinKitChasingDots(
-      color: Theme.of(context).textTheme.headline6.color,
-      size: size,
-    );
+    if(this.loader == null) {
+      return new SpinKitChasingDots(
+        color: Theme.of(context).textTheme.headline6.color,
+        size: size,
+      );
+    }
+    else return this.loader;
   }
 
   factory Loading.small() {
     return Loading(size: 15);
   }
 
+  factory Loading.threeBounce({@required Color color, double size = 30}) {
+    return Loading(
+      loader: SpinKitThreeBounce(
+        color: color,
```

Comment:
I don't know if the color is an expected configuration we want to be set externally.
`Loading` widget could have an enum to define the type of the loader to use?
`CHASING_DOTS` or `THREE_BOUCE` ?

Reply:
I used this solution which was the cleanest solution of the two (a color of the Theme can be used because we can access the context).